install-local.ps1

```diff
@@ -0,0 +1,4 @@
+dotnet build $PSScriptRoot\src\dgt.power\dgt.power.csproj
+dotnet pack $PSScriptRoot\src\dgt.power\dgt.power.csproj
```

Comment:
We can delete this file it doesn't work anymore, already purged it from the last release